src/wms/src/test/java/org/geoserver/wms/featureinfo/GetFeatureInfoJSONTest.java

```diff
@@ -249,7 +249,7 @@ public void testPropertySelection() throws Exception {
         MockHttpServletResponse response = getAsServletResponse(request, "");
 
         // MimeType
-        assertEquals(JSONType.json, response.getContentType());
+        assertEquals(JSONType.json, getMimeTypeAndCharacterEncoding(response.getContentType())[0]);
```

Comment:
It seems that (almost?) everywhere the call is ``getMimeTypeAndCharacterEncoding(response.getContentType())[0])``... then it would be easier to have a simpler ``getMimeType`` call, maybe with two overrides, one taking a string, another taking a response object.

Reply:
They all use the first index, so switched to returning the mime type as string only.